src/api/content.js

```diff
@@ -1,17 +1,45 @@
 export const content = [
-  { name: "MILK", alt: ["SOY MILK", "ALMOND MILK", "OAT MILK"] },
-  { name: "EGG", alt: "CHIA EGG" },
-  { name: "BUTTER", alt: "MARGERINE" },
-  { name: "CREAM", alt: "COCONUT CREAM" },
-  { name: "BUTTERMILK", alt: "" },
-  { name: "CHEESE SLICES", alt: "" },
-  { name: "GELATIN", alt: "" },
-  { name: "SOUR CREAM", alt: "" },
-  { name: "BACON", alt: "" },
-  { name: "YOGHURT", alt: "" },
-  { name: "CHICKEN STOCK", alt: "" },
-  { name: "BEEF STOCK", alt: "" },
-  { name: "MAYONNAISE", alt: "" },
-  { name: "SCRAMBLED EGG", alt: "" },
-  { name: "CONDENSED MILK", alt: "" }
+  "Milk",
+  "Egg",
+  "Butter",
+  "Cream",
+  "Buttermilk",
+  "Cheese Slices",
+  "Gelatin",
+  "Sour Cream",
+  "Bacon",
+  "Scrambled Egg",
+  "Yoghurt",
+  "Mayonnaise",
+  "Chicken Stock",
+  "Beef Stock",
+  "Condensed Milk",
+  "Worcestershire Sauce",
+  "Parmesan",
+  "Cream Cheese",
+  "Milk Chocolate",
+  "Honey",
+  "Sugar",
+  "Pulled Pork",
+  "Pulled Chicken",
+  "Salmon",
+  "Ground Beef"
 ];
+
+// export const content = [
```

Comment:
I was here before Marwin!! no dead code :D

Reply:
...BUT I might need it again! 🙈